Makefile

```diff
@@ -29,6 +29,10 @@ VPP_AGENT?=ligato/vpp-agent:v3.0.1
 
 include build/nse/nses.mk
 
+test-e2e:
+	@echo "Running e2e tests.."
+	./test/e2e/run_e2e.sh
```

Comment:
IMO, we should default to pin VPP_IMG version in the makefile but allow it to be overriden, e.g. "VPP_IMG ?= 20.X"